common.js

```diff
@@ -34,7 +34,7 @@ function insertInfoPane() {
         // contact info
         html += '<div id="sidebar-contact">';
         html += '<h3>Contacts</h3>';
-        html += '<p>TSC Chair: <a href="mailto:ken.museth@gmail.com">ken.museth@gmail.com</a> </br> Developers: </br><a href="mailto:openvdb@gmail.com">openvdb@gmail.com</a> </br> Discussion Forum:</br><a href="http://www.openvdb.org/forum/">openvdb-forum@googlegroups.com</a></p>';
+        html += '<p>TSC Chair: <a href="mailto:ken.museth@gmail.com">ken.museth@gmail.com</a> </br> Developers: </br><a href="mailto:openvdb@gmail.com">openvdb@gmail.com</a> </br> Discussion Forum:</br><a href="http://www.openvdb.org/forum/">openvdb-forum@googlegroups.com</a> </br> Private Security Alias: </br><a href="mailto:security@openvdb.org">security@openvdb.org</a></p>';
```

Comment:
Is "Private Security Alias" standard terminology?  How about calling it "Security Concerns" or "Security Issues"?

It would probably also be good to split up that very long line.

Reply:
I've renamed it "Security Issues (Private)", thought it was still useful to make it known that this wasn't an open email list.

I also refactored the line length and changed the developers contact from openvdb@gmail.com to the aswf openvdb-dev list and renamed it TSC Members (I'm not sure who has access to openvdb@gmail.com). 

I note that the CLA email address is still openvdb@gmail.com - should that also be changed, perhaps to the private openvdb aswf list?

Reply:
I understand that you're trying to convey that security-related communications are confidential, but labeling it "Private" suggests (to me, at least) that it's for internal use only.

Reply:
That's a fair point. I guess I'll just make it "Security Issues" then, unless you have any better suggestions?

Reply:
I would go with just "Security Issues".  The Security Policy document makes it clear that it's confidential.

Reply:
Sure, just done that. Thanks!